content/docs/gke/authentication.md

```diff
@@ -0,0 +1,170 @@
+```
+
+Roles can also be viewed and modified through the 
+[GCP IAM console](https://console.cloud.google.com/iam-admin/).
+
+
+More information about IAM can be found in the 
+[GCP docs](https://cloud.google.com/iam/docs/granting-changing-revoking-access).
+
+## Authenticating kubectl
+The [`kubectl` tool](https://kubernetes.io/docs/reference/kubectl/overview/) is used for interacting with a Kubernetes cluster through the command line
+
+### Authorizing with a GCP Account
+If you set up your Kubernetes cluster using GKE, you can authenticate with the cluster using a GCP account. 
+The following command will fetch the credentials for your cluster and save them to your local 
+[`kubeconfig` file](https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/):
+
+```
+CLUSTER_NAME=your-gke-cluster
+ZONE=your-gcp-zone
+
+gcloud container clusters get-credentials $CLUSTER_NAME --zone $ZONE
+```
+
+More information can be found in the 
+[GCP docs](https://cloud.google.com/kubernetes-engine/docs/how-to/cluster-access-for-kubectl).
+
+### Changing Active Clusters
+If you work with multiple Kubernetes clusters, you may have multiple contexts saved in your local 
+[`kubeconfig` file](https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/).
+To view the clusters you have saved, run the following command:
+
+```
+kubectl config get-contexts
+```
+
+This will show which cluster is currently being controlled by your `kubectl` commands.
+To change your active cluster, run the following command:
+```
+CONTEXT_NAME=your-new-context
+
+kubectl config set-context $CONTEXT_NAME
+```
+
+More information can be found in the 
+[Kubernetes docs](https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/).
+
+### Checking RBAC Permissions
+
+Like GKE IAM, Kubernetes permissions are typically handled with a "role-based authorization control" (RBAC) system.
+Each Kubernetes service account has a set of authorized roles associated with it. If your account doesn't have the 
+right roles assigned to it, certain tasks will fail.
+
+To check if an account has the proper permissions to run a command, you can build a query structured as
+`kubectl auth can-i [VERB] [RESOURCE] --namespace [NAMESPACE]`. For example, the following command will verify
+that your account has permissions to create deployments in the `Kubeflow` namespace:
+, 
+```
+kubectl auth can-i create deployments --namespace kubeflow
+```
+
+More information can be found in the 
+[Kubernetes docs](https://kubernetes.io/docs/reference/access-authn-authz/authorization/).
+
+### Adding RBAC Permissions
+If you find you are missing a permission you need, you can grant the missing roles to your service account using
+Kubernetes resources.
+- **Roles** describe the permissions you want to assign. For example, `verbs: ["create"], resources:["deployments"]`
```

Comment:
If you add a new line before roles, the list will render?

Reply:
good catch